core/application/configuration_storage.hpp

```diff
@@ -22,6 +25,21 @@ namespace kagome::application {
      * @return genesis block of the chain
      */
     virtual const primitives::Block &getGenesis() const = 0;
+
+    /**
+     * Return ids of peer nodes of the current node
+     */
+    virtual std::vector<libp2p::peer::PeerId> getPeerIds() const = 0;
+
+    /**
+     * Return peers' session keys used in BABE
+     */
+    virtual std::vector<crypto::SR25519PublicKey> getSessionKeys() const = 0;
+
+    /**
+     * Return public keys of authority nodes
+     */
+    virtual std::vector<crypto::ED25519PublicKey> getAuthorities() const = 0;
```

Comment:
we need to add authority id for current node, which is its public session key crypto::ED25519PublicKey.

Reply:
Sorry, add where?

Reply:
dismiss it, we already solved this question